source/patterns/@aws-solutions-constructs/core/test/test-helper.ts

```diff
@@ -178,3 +177,26 @@ export function expectNonexistence(stack: Stack, type: string, props: object) {
   const shouldFindNothing = Template.fromStack(stack).findResources(type, props);
   expect(Object.keys(shouldFindNothing).length).toEqual(0);
 }
+
+// private helper class to suppress the standard cfn nag warnings for lambda functions used in integ tests
+class CfnNagLambdaAspect implements IAspect {
+  public visit(node: IConstruct): void {
+    const resource = node as CfnResource;
+    if (resource.cfnResourceType === 'AWS::Lambda::Function') {
+      addCfnSuppressRules(resource, [
+        { id: 'W58', reason: 'CDK generated custom resource' },
```

Comment:
Is this for lambda functions in tests or for custom resources? This seems a material difference...

Reply:
This is for lambda functions in tests, which is why I put it in the `test-helper.ts`

Reply:
I got you now, I can update the suppression reason's to indicate its a test resource :)

Reply:
done.